cli/src/cli.rs

```diff
@@ -38,6 +38,10 @@ pub struct RunCmd {
     #[structopt(long, default_value = "10000")]
     pub max_past_logs: u32,
 
+    /// Maximum fee history cache size.
```

Comment:
The above `pub sealing: Sealing,` doesn't contain definition of `Sealing`,  you may want to add the definition or remove this field.
![image](https://user-images.githubusercontent.com/4054836/149766806-ccf69307-04e2-4062-9184-f1d08330e5a3.png)

Reply:
Removed. Thanks.